README.md

```diff
@@ -182,6 +182,100 @@ curl http://127.0.0.1:8080/gotify_webhook -d '
 ]}
 '
 ```
+### Bridge Message Templating
+The bridge now supports user-defined templating for all inbound messages. The user-defined templating can be used for the title and/or message. Also, user-defined templating overrides the default title and message annotations. All keys and values in the JSON from alertmanager can be used in the user-defined template. Any failures in the templates will result in the bridge defaulting back to default alerting.
+
+#### Usage Notes:
+- For Docker, you must bind your volume to your host to add user-defined templating.
+- The default directory for all templates is the root of the bridge in the folder called `templates`.
+- User-defined templating allows matching and linking using the "define" name of the template [Go Templating](https://golang.google.cn/pkg/text/template/).
+- The Gotify software token is used for matching a message template.
+- The Gotify software token with `title=` at the beginning is used for matching a title template.
```

Comment:
When working with "defined" templates, something within the "defined" name must be set to determine a match when executing a template. This is the best option when using a more open/unrestricted folder structure because no file name reads are required. We could add a check, but the detected templates with a title would need to be tracked so know the defined template with the token is a title vs. a message.

Reply:
OK, I think I follow. I'm still fairly new to the world of templates (I know how to update a single template that already exists or to operate on a single template, but the rest requires spelunking source code to wrap my head around it). Maybe the wording of the statement just threw me off. Would it be better to maybe say something along the lines of "If a template is defined as `title=TOKEN`, it will be used to render the title of the alert" (depending how the folder structure convo pans out)

Reply:
Yeah, I think that sounds better. If you were a little thrown off, it would have also caused it to others. I am submitting that ReadMe revision now.